docs/user_guide/python_api.md

```diff
@@ -1,44 +1,42 @@
-.. _python_api:
+(python-api)=
 
-==========
-Python API
-==========
+# Python API
 
-Basic Usage
-===========
+## Basic Usage
 
 JupyterGIS provides a Python API that can be used for opening QGIS projects,
 add raster or vector layers and apply filtering.
 
 You can open an existing QGIS project and **display it in your Jupyter Notebook**:
 
-.. code-block:: Python
+```Python
+from jupytergis import GISDocument
 
-    from jupytergis import GISDocument
+doc = GISDocument('file.qgz')
 
-    doc = GISDocument('file.qgz')
-
-    doc
+doc
+```
 
 **Opening an existing file connects you to the file’s collaborative session**,
 meaning that anyone working on the same QGIS project file, whether through the
 JupyterLab extension or the Python API, will see the edits you make.
 
-Creating a ``GISDocument`` object without providing a path to an existing file would create a
+Creating a `GISDocument` object without providing a path to an existing file would create a
 new empty document:
 
-.. code-block:: Python
-
-    from jupytergis import GISDocument
+```Python
+from jupytergis import GISDocument
 
-    doc = GISDocument()
+doc = GISDocument()
 
-    doc
+doc
+```
 
 Once the document is opened/created, you can start creating GIS layers.
 
-``GISDocument`` API Reference
-=============================
+## `GISDocument` API Reference
 
+```{eval-rst}
```

Comment:
I believe we have to keep the `eval-rst` directive here: https://myst-parser.readthedocs.io/en/stable/syntax/code_and_apis.html#sphinx-ext-autodoc